src/view/com/auth/onboarding/RecommendedFeeds.tsx

```diff
@@ -130,6 +131,7 @@ export function RecommendedFeeds({next}: Props) {
               renderItem={({item}) => <RecommendedFeedsItem item={item} />}
               keyExtractor={item => item.uri}
               style={{flex: 1}}
+              showsVerticalScrollIndicator={false}
```

Comment:
These screens actually look better in general without indicators (and already do not show them on web) so let's just set the prop to false.